server/aws-lsp-codewhisperer/src/language-server/codeWhispererSecurityScanServer.ts

```diff
@@ -67,12 +95,17 @@ export const SecurityScanServerToken =
                     throw error
                 } finally {
                     await dependencyGraph.removeTmpFiles()
+                    securityScanTelemetryEntry.artifactsUploadDuration = performance.now() - uploadStartTime
                 }
+                scanHandler.throwIfCancelled(token)
                 /**
                  * Step 3:  Create scan job
                  */
+                serviceInvocationStartTime = performance.now()
                 const scanJob = await scanHandler.createScanJob(artifactMap, document.languageId.toLowerCase())
-                logging.log(`Created security scan job.`)
+                logging.log(`Created security scan job id: ${scanJob.jobId}`)
+                securityScanTelemetryEntry.codewhispererCodeScanJobId = scanJob.jobId
+                scanHandler.throwIfCancelled(token)
```

Comment:
Do we need this here? We are already checking on line 117 just after jobStatus check.

Reply:
Yes, missed the pollScanJobStatus method call.